src/references/binding_sets.md

```diff
@@ -0,0 +1,266 @@
+
+![!Bindings Manager](../img/BindingSets/bindings_manager_window.png){: .center loading="lazy" }
+
+## Creating a Binding Set
+
+A **Binding Set** needs to be placed into the **Hierarchy**. A new **Binding Set** will not overwrite an existing **Binding Set** that is in the **Hierarchy**. For example, if one **Binding Set** has ++Space++ set to perform the **Jump** action, and another **Binding Set** has ++Space++ set for to perform the **Crouch** action, then both actions will be performed.
+
+!!! warning "Placing a **Binding Set** in a **Server Context** will not work in local multiplayer or a published game."
+
+### Bindings Manager
+
+A **Binding Set** can be created from the **Bindings Manager** window by clicking on the button **Create new Binding Set**. When a **Binding Set** is created, it will be added to **My Binding Sets** in **Project Content**.
+
+![Create Button](../img/BindingSets/create_binding_set.png){: .center loading="lazy" }
+
+**Binding Sets** created this way will have the property **Binding Sets** under **Bindings** already setup with the asset reference when placed into the **Hierarchy**.
+
+![!Asset Reference](../img/BindingSets/asset_reference.png){: .center loading="lazy" }
+
+### Binding Set Component
+
+A **Binding Set** can be created by dragging the **Binding Set** component from **Gameplay Objects** under **GAME OBJECTS** into the **Hierarchy**. This object will not have the **Binding Set** asset reference created, the creator would need to create a new set of bindings, and drag the asset onto the property.
+
+![!Core Content](../img/BindingSets/core_content.png){: .center loading="lazy" }
+
+## Adding a Binding
+
+A binding can be a specific key, button, or directional control that performs an action.
+
+Adding a binding to a **Binding Set** can be done from the **Bindings Manager** window by clicking on **Add Binding**.
+
+![!Add Binding](../img/BindingSets/add_binding.png){: .center loading="lazy" }
+
+For example, setting the key ++Q++ to perform the **Jump** action.
+
+1. Add **Basic Binding**.
+2. Select **Jump** from the **Core Behavior** drop down.
+3. Set the letter ++Q++ from the **Keyboard Primary** drop down.
+
+![!Jump Action](../img/BindingSets/jump_action.png){: .center loading="lazy" }
+
+When a new binding is added, the action name and description can be set. The action name should be something that identifies what this action does. This identifier can be used in Lua to determine which action triggered the event.
+
+Bindings support a secondary key and controller input. Meaning that the player can perform an action using 2 different keys or use controller input (i.e. a button).
+
+### Core Behavior
+
+When creating a binding, a **Core Behavior** needs to be selected from the drop down so that the **Binding Set** knows which action to perform when the binding is triggered.
+
+![!Core Behavior](../img/BindingSets/core_behaviors.png){: .center loading="lazy" }
+
+!!! info "Binding Defaults"
```

Comment:
This, for example, is important enough that it should be in the main text. If this were a tutorial, the info box would be a perfect aside, but because it's central to the purpose of this doc I would keep it with the main text